context/app/static/js/shared-styles/dropdowns/DropdownListbox/style.js

```diff
@@ -0,0 +1,14 @@
+import styled from 'styled-components';
+import Popper from '@material-ui/core/Popper';
+import Paper from '@material-ui/core/Paper';
+
+const StyledPopper = styled(Popper)`
+  z-index: 50;
```

Comment:
Do you think it would be good to have a central definition of z-index values? Or do you know of other good practices? I'm imaging a file with a list of integer consts, so as we're adding a new rule, it's clear what it's above or below.

This might be overkill.

Reply:
I think that makes sense either putting the z-index values in the theme or a separate file which can be imported in the style.js files.

Reply:
Yeah: Something like that. Doesn't need to be this PR.

Reply:
Filed #1340